### ISSUE TITLE
Sluggish seek and sometimes play/pause

### ISSUE THREAD

**Issue by Sorgair** [mifi/lossless-cut#1197]
This starts to happen between version 3.7 and 3.8, so it is from one of the new features added in 3.8 I assume. It happens on all versions since 3.8 as well, including the latest version, on all files; I am using mouse and keyboard to seek.

When clicking somewhere on the timeline to seek, it takes between instant to 3ish seconds to seek, whereas before in version 3.7 it is always instant. I suspect it has to do with the keyframes being shown on the timeline, as anything that makes the timeline look different (such as selecting a start/end keyframe) usually lags the UI. 

I have tried using both the .zip and .exe download.

I'm on windows 10, and my pc specs are good enough that seeking shouldn't randomly be slow.

**Comment by mifi** [comment]
just to be sure, you mean this version from 2020, right? https://github.com/mifi/lossless-cut/releases/tag/v3.8.0

could be related to #1097 but I'm baffled by you experiencing this on such old versions

**Comment by Sorgair** [comment]
yeah it's that version, I went back to when the keyframe timeline was first added to find it

since I think I'm the only one with this, it could just be I have some problem with my OS since I haven't reset it in 6 years

also, watching videos of other people using lossless cut, mine takes far longer to open, where opening it from windows search hangs for ~3 seconds, leaving the search menu open, then for the first ~10ish seconds the window is just white. also, some other things show up very slowly, for example, the sunflowers appear for me 2 seconds after everything else

**Comment by mifi** [comment]
> also, watching videos of other people using lossless cut, mine takes far longer to open, where opening it from windows search hangs for ~3 seconds, leaving the search menu open, then for the first ~10ish seconds the window is just white. also, some other things show up very slowly, for example, the sunflowers appear for me 2 seconds after everything else

are you using the .exe download or the .zip?

**Comment by Sorgair** [comment]
it happens on both

**Comment by mifi** [comment]
which os? is your computer very old? maybe you can try to delete the losslesscut directory from appdata (note you will loose all settings)

**Comment by Sorgair** [comment]
i tried deleting the directory but it didnt work 

I'm on windows 10, it could be and probably is because I haven't reinstalled windows in 6 years, but everything is up to date

**Comment by mifi** [comment]
can you try with the newest version now?

**Comment by mifi** [comment]
@DGrv hi. just to be sure, you're using an external version of ffmpeg and it makes seeking faster? And if you disable external ffmpeg again seeking becomes slow? I don't understand how that can happen. Care to record a screencast that shows the problem?

**Comment by DGrv** [comment]
Mmmmm, seems that I do not succeed to reproduce it ... really strange.
After disabling external ffmpeg again it is staying smooth. I do not understand it neither now.
The 2 open points on my side are now the raw video from my camera (unfortunately I do not have anymore raw video from my gopro since I got it stolen few days ago) and that I updated few drivers the last days (after the last post) and the driver for the video card was I think one of them. 

So my last post may be just bull shit but at least people can try ...

I will test with new videos from someone else with the same camera and same settings and if I can anyhow reproduce this behavior I will do for sure a screencast. Sorry for this.
